config/key.go

```diff
@@ -19,6 +19,13 @@ type KeyPair struct {
 	Secret abstract.Secret // Secret key
 }
 
+// NewKeyPair directly creates a secret/public key pair
+func NewKeyPair(suite abstract.Suite) (abstract.Secret, abstract.Point) {
```

Comment:
Maybe NewRandomKeyPair would be better suited, but I know that there's some minimalistic-struct-name coder out there so ...
Otherwise, looks fine for me and useful !